svgdigitizer/electrochemistry/cv.py

```diff
@@ -206,72 +265,102 @@ def axis_properties(self):
             ... <svg>
             ...   <g>
             ...     <path d="M 0 200 L 0 100" />
-            ...     <text x="0" y="200">x1: 0 V vs. RHE</text>
+            ...     <text x="0" y="200">E1: 0 V vs. RHE</text>
             ...   </g>
             ...   <g>
             ...     <path d="M 100 200 L 100 100" />
-            ...     <text x="100" y="200">x2: 1 V vs. RHE</text>
+            ...     <text x="100" y="200">E2: 1 V vs. RHE</text>
             ...   </g>
             ...   <g>
             ...     <path d="M -100 100 L 0 100" />
-            ...     <text x="-100" y="100">y1: 0 uA / cm2</text>
+            ...     <text x="-100" y="100">j1: 0 uA / cm2</text>
             ...   </g>
             ...   <g>
             ...     <path d="M -100 0 L 0 0" />
-            ...     <text x="-100" y="0">y2: 1  uA / cm2</text>
+            ...     <text x="-100" y="0">j2: 1 uA / cm2</text>
             ...   </g>
             ...   <text x="-200" y="330">scan rate: 50 V/s</text>
             ... </svg>'''))
             >>> cv = CV(SVGPlot(svg))
-            >>> cv.axis_properties
-            {'x': {'dimension': 'E', 'unit': 'V'}, 'y': {'dimension': 'j', 'unit': 'A / m2'}}
+            >>> cv.current_dimension
+            'j'
 
         """
-        return {
-            "x": {"dimension": "E", "unit": "V"},
-            "y": {
-                "dimension": "j"
-                if "m2"
-                in str(CV.get_axis_unit(self.svgplot.axis_labels[self.svgplot.ylabel]))
-                else "I",
-                "unit": "A / m2"
-                if "m2"
-                in str(CV.get_axis_unit(self.svgplot.axis_labels[self.svgplot.ylabel]))
-                else "A",
-            },
-        }
+        dimensions = list(set(["I", "j"]).intersection(self.svgplot.schema.field_names))
 
-    @classmethod
-    def get_axis_unit(cls, unit):
-        r"""
-        Return `unit` as an `astropy <https://docs.astropy.org/en/stable/units/>`_ unit.
+        if len(dimensions) == 1:
+            if self.svgplot.schema.get_field(dimensions[0])["orientation"] == "y":
+                return dimensions[0]
+            raise Exception("The current must be on the x-axis.")
+
+        raise Exception("No current axis or more than one current axis found.")
```

Comment:
The same as above for the voltage axis, except that currents are only allowed on the y-axis.